src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1151,4 +1151,22 @@ Win + Shift + O to toggle your video</value>
   <data name="SettingsWindow_Title" xml:space="preserve">
     <value>PowerToys Settings</value>
   </data>
+  <data name="Position_GroupSettings.Text" xml:space="preserve">

```

Comment:
Is this used anywhere? I can't find a reference to it.

Reply:
Ah woops. Left over from the rebase; I've removed again.